homeassistant/components/demo/stt.py

```diff
@@ -0,0 +1,60 @@
+"""Support for the demo for speech to text service."""
+from typing import List
+
+from aiohttp import StreamReader
+
+from homeassistant.components.stt import Provider, SpeechMetadata, SpeechResult
+from homeassistant.components.stt.const import (
+    AudioBitrates,
+    AudioFormats,
+    AudioSamplerates,
+    AudioCodecs,
+    SpeechResultState,
+)
+
+SUPPORT_LANGUAGES = ["en", "de"]
+
+
+async def async_get_engine(hass, config):
+    """Set up Demo speech component."""
+    return DemoProvider()
+
+
+class DemoProvider(Provider):
+    """Demo speech API provider."""
+
+    @property
+    def supported_languages(self) -> List[str]:
+        """Return a list of supported languages."""
+        return SUPPORT_LANGUAGES
+
+    @property
+    def supported_formats(self) -> List[AudioFormats]:
+        """Return a list of supported formats."""
+        return [AudioFormats.WAV]
+
+    @property
+    def supported_codecs(self) -> List[AudioCodecs]:
+        """Return a list of supported codecs."""
+        return [AudioCodecs.PCM]
+
+    @property
+    def supported_bitrates(self) -> List[AudioBitrates]:
```

Comment:
```suggestion
    def supported_bit_rates(self) -> List[AudioBitRates]:
```

Reply:
bitrate is 1 word

![image](https://user-images.githubusercontent.com/1444314/68063018-45231b00-fccb-11e9-964e-ed73befff221.png)

Reply:
It can be one word but I suggest we go with two for consistency with sample rate.
https://www.merriam-webster.com/dictionary/bit%20rate